Modified ko-KR.json

NT-FM opened the pull request:
Fixed some words, awkward points, and consistency

saengjuwa commented:
fianlly

RedLime commented:
"스타트포스"가 인게임 상에서 영문으로 "Start Pos"로 적혀있는 상태의 오브젝트라서, 스타트포스라고 하면 혼동이 올 수 있을 것 같기도 합니다.

StartPos와 같이 인게임 내에서 직접적으로 영문으로 표시되어있는 오브젝트들은 영문 그대로 냅두어야한다고 생각하는데, 다른 분들 의견도 좀 궁금하네요.

BlueTomcat1 commented:
영어 음차가 통용되지 않거나 가독성을 해치지 않는다면 굳이 바꿀 이유는 없다고 봅니다. 실제 사용할때도 굳이 'StartPos'와 같이 영어로 쓰기보다는 '스타트포스'로 쓰는 경향이 많기도 하고요. 
추가로 지금 "Normal Mode"가 "일반 모드"로 번역되어 있는데, 일반 모드는 기본 크기 상태를 주로 지칭하는 경향이 있다고 생각해서 차라리 Normal을 그대로 음차한 "노말 모드"가 낫지 않나 싶습니다.

absoIute commented:
Please resolve conflicts with the current Korean translation and commit any changes you deem appropriate if you want this merged!

saengjuwa commented:
그 리퀘스트 올린사람은 제가 아니라서

2024년 12월 8일 (일) 오후 10:20, RedLime ***@***.***>님이 작성:

> "스타트포스"가 인게임 상에서 영문으로 "Start Pos"로 적혀있는 상태의 오브젝트라서, 스타트포스라고 하면 혼동이 올 수 있을
> 것 같기도 합니다.
>
> StartPos와 같이 인게임 내에서 직접적으로 영문으로 표시되어있는 오브젝트들은 영문 그대로 냅두어야한다고 생각하는데, 다른 분들
> 의견도 좀 궁금하네요.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/absolllute/Mega-Hack-Translate/pull/40#issuecomment-2525868878>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLGUVTLAAM3HHB6UDDHZ3EL2ERBTPAVCNFSM6AAAAABTHCGAPSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKMRVHA3DQOBXHA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

saengjuwa commented:
네 알겠습니자

2024년 12월 8일 (일) 오후 11:38, 와생주 ***@***.***>님이 작성:

> 그 리퀘스트 올린사람은 제가 아니라서
>
> 2024년 12월 8일 (일) 오후 10:20, RedLime ***@***.***>님이 작성:
>
>> "스타트포스"가 인게임 상에서 영문으로 "Start Pos"로 적혀있는 상태의 오브젝트라서, 스타트포스라고 하면 혼동이 올 수 있을
>> 것 같기도 합니다.
>>
>> StartPos와 같이 인게임 내에서 직접적으로 영문으로 표시되어있는 오브젝트들은 영문 그대로 냅두어야한다고 생각하는데, 다른 분들
>> 의견도 좀 궁금하네요.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/absolllute/Mega-Hack-Translate/pull/40#issuecomment-2525868878>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BLGUVTLAAM3HHB6UDDHZ3EL2ERBTPAVCNFSM6AAAAABTHCGAPSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKMRVHA3DQOBXHA>
>> .
>> You are receiving this because you commented.Message ID:
>> ***@***.***>
>>
>

saengjuwa commented:
아니 이게 깃허브 메세지였구나

2024년 12월 8일 (일) 오후 11:38, 와생주 ***@***.***>님이 작성:

> 네 알겠습니자
>
> 2024년 12월 8일 (일) 오후 11:38, 와생주 ***@***.***>님이 작성:
>
>> 그 리퀘스트 올린사람은 제가 아니라서
>>
>> 2024년 12월 8일 (일) 오후 10:20, RedLime ***@***.***>님이 작성:
>>
>>> "스타트포스"가 인게임 상에서 영문으로 "Start Pos"로 적혀있는 상태의 오브젝트라서, 스타트포스라고 하면 혼동이 올 수
>>> 있을 것 같기도 합니다.
>>>
>>> StartPos와 같이 인게임 내에서 직접적으로 영문으로 표시되어있는 오브젝트들은 영문 그대로 냅두어야한다고 생각하는데, 다른
>>> 분들 의견도 좀 궁금하네요.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/absolllute/Mega-Hack-Translate/pull/40#issuecomment-2525868878>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/BLGUVTLAAM3HHB6UDDHZ3EL2ERBTPAVCNFSM6AAAAABTHCGAPSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKMRVHA3DQOBXHA>
>>> .
>>> You are receiving this because you commented.Message ID:
>>> ***@***.***>
>>>
>>

NT-FM commented:
Modified the file to a new one based on the current translation file.
Fixed the major translation error/consistency.
특정 트리거를 비활성화시키는 설정의 경우 영어 원문을 그대로 표기하여 어떠한 트리거가 비활성화되는지 정확히 알 수 있도록 알려 주고, 스타트포스의 경우 음차와 표기의 괴리가 거의 없으며 보편적으로 통용되는 용어기에 한글로 표기했습니다.
'일반 모드'라는 말이 쓰인 문단이 연습 모드를 종료하기 전에 팝업창을 띄우는 기능을 활성화시키는 설정을 가르키는 문단이기에 이를 크기 상태와 헷갈릴 가능성은 사실상 없으므로 '일반 모드'로 표기하여도 큰 문제가 없다 봅니다.
